node/libs/crypto/src/lib.rs

```diff
@@ -1,8 +1,8 @@
 //! Collection of cryptographic primitives used in zksync-bft repository.
 
-pub mod bls12_381;
```

Comment:
Don't delete the module. Let's keep it here.

Reply:
Fixed.